CMakeLists.txt

```diff
@@ -132,6 +132,7 @@ if(RPM_FOUND)
 	check_library_exists("${RPM_LIBRARY}" headerFormat "" HAVE_HEADERFORMAT)
 	check_library_exists("${RPMIO_LIBRARY}" rpmFreeCrypto "" HAVE_RPMFREECRYPTO)
 	check_library_exists("${RPM_LIBRARY}" rpmFreeFilesystems "" HAVE_RPMFREEFILESYSTEMS)
+	check_library_exists("${RPM_LIBRARY}" rpmVerifyFile "" HAVE_RPMVERIFYFILE)
```

Comment:
There is a warning on RHEL7:
```
/home/jenkins/workspace/openscap-pull-requests-multi/label/rhel7/src/OVAL/probes/unix/linux/rpm-helper.c:39:5: warning: implicit declaration of function ‘rpmfiVerify’ [-Wimplicit-function-declaration]
     rpmVerifyAttrs vfy = rpmfiVerify(fi, omitMask);
     ^
```

and error:

```
Linking C executable oscap
../src/libopenscap.so.25.0.0: undefined reference to `rpmfiVerify'
collect2: error: ld returned 1 exit status
gmake[2]: *** [utils/oscap] Error 1
```


I think you need to add `#cmakedefine HAVE_RPMVERIFYFILE` to `config.h.in` around line 42 to fix this.